src/modules/launcher/PowerLauncher/MainWindow.xaml.cs

```diff
@@ -290,6 +290,26 @@ private string ListView_FirstItem(String input)
 
             return string.Empty;
         }
+
+        private string GetSearchText(String input, string query)
+        {
+            if (!string.IsNullOrEmpty(input))
+            {
+                string selectedItem = _viewModel.Results?.SelectedItem?.ToString();
+                int selectedIndex = _viewModel.Results.SelectedIndex;

```

Comment:
if viewmodel.Results is null, then we would get an exception because it is directly trying to access selectedIndex. This can be mitigated using `_viewModel.Results?.SelectedIndex`.